docs/book/developer-guide/stacks-profiles-repositories/repository.md

```diff
@@ -107,7 +107,7 @@ ZenML repository initialized at /tmp/zenml.
 The local active profile was initialized to 'default' and the local active stack
 to 'default'. This local configuration will only take effect when you're running
 ZenML from the initialized repository root, or from a subdirectory. For more
-information on profile and stack configuration, please visit https://docs.zenml.io.
+information on profile and stack configuration, please visit https://docs.zenml.io .
```

Comment:
Was this change intentional? If not, might want to undo:
```suggestion
information on profile and stack configuration, please visit https://docs.zenml.io.
```

Reply:
Yes I though that looks a little awkward too.